jjb/dynamic/scale-ci-ms-osde2e.yml

```diff
@@ -232,12 +226,57 @@
             - ERROR
             - WARNING
             - DEBUG
+      - bool:
+          default: true
```

Comment:
@morenod Shouldn't this be false if our default behavior is not to cleanup the clusters? Same with rosa one below

Reply:
yep, updated on both rosa and osde2e